.github/workflows/gb-docs.yml

```diff
@@ -0,0 +1,20 @@
+name: GB Docs Link Checker
+
+on:
+  schedule:
+    - cron: '0 0 * * *'  # every day at midnight UTC
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - v2
+
+jobs:
+  lychee-check:
```

Comment:
I suggest we run this on PR merges that modify files in `docs-gb` (using something like the [paths-filter action](https://github.com/dorny/paths-filter)), and only run on cron much less frequent (say, once every two weeks) to detect stale external links.